src/effects/audiounits/AudioUnitEffect.h

```diff
@@ -77,8 +77,8 @@ class AudioUnitEffect : public wxEvtHandler,
 
    // EffectProcessor implementation
 
-   unsigned GetAudioInCount() override;
-   unsigned GetAudioOutCount() override;
+   unsigned GetAudioInCount() const override;
+   unsigned GetAudioOutCount() const override;
```

Comment:
One remaining small defect of this pull request is that the first commit fails to build on MacOS but the second commit corrects that.  The declaration of GetAudioInCount does not match the definition in the first commit.

I trust you to fix that as you rebase into latest master.

Reply:
That's right! thanks, corrected.